_layouts/meltano_plugin.html

```diff
@@ -326,21 +295,47 @@ <h3 id="next-steps">Next steps</h3>
 
 <h2 id="capabilities">Capabilities</h2>
     
-{% if variant.capabilities.size == 0 %}
+{% if plugin.capabilities.size == 0 %}
 <p>
-  Check the
-  <a href="{{variant.repo}}#readme" target="_blank">README and code in the repository</a>
-  for more information on capabilities for this {{plugin_type}}.
+  <code>{{plugin.name}}</code> does not have any capabilities defined in its metadata.
+  Please considering adding them by making a PR to the
+  <a href="{{yaml_file}}">YAML file</a>
+  that defines the capabilities for this {{plugin_type}}.
```

Comment:
Can we (now or soon after 2.0) link to a doc describing what capabilities are, and how to find out which ones a tap/target supports? The closest thing we have today is https://docs.meltano.com/guide/integration#validate-tap-capabilities and https://docs.meltano.com/contribute/plugins#how-to-test-a-tap, but neither are as clear as they could be.

Edit: I just found https://hub.meltano.com/singer/docs#singer-connector-capabilities! That's not super clear either though, since "Log Based" for example isn't a capability keyword. And `discovery` should be `discover`, shouldn't it?